### ISSUE TITLE
Username length affects how much text fits in a line on v2

### ISSUE THREAD

**Issue by Xinnoh** [ppy/osu-web#2502]
The length of the username affects how much space the user has before text wraps. Users with short names will be able to put more text in one line, long names require more paragraphs

Current - https://osu.ppy.sh/beatmapsets/736560/discussion/-/generalAll#/111472
![screen shot 2018-02-20 at 9 18 36 am](https://user-images.githubusercontent.com/30382015/36403300-13739b46-161f-11e8-93a2-1ce81c53d095.png)

Suggestion
![screen shot 2018-02-20 at 9 18 36 am](https://user-images.githubusercontent.com/30382015/36403403-b2562fb2-161f-11e8-90f7-d15c561ab06a.png)

**Comment by dorothy3242** [comment]
This is a problem for those with the special tags like "Nominator" too, it's so long it shortens line space even if you had a short name before. This could also go under the suggested position for usernames? Maybe?
![image](https://user-images.githubusercontent.com/33286824/36403594-d8809dd8-15b3-11e8-8c74-29e00b9fd7d1.png)

**Comment by Xinnoh** [comment]
Yeah, Sotarks' mapper role was cut off due to my crop, but it would have been underneath his name

**Comment by peppy** [comment]
At very least I really do think there needs to be alignment of the username area.

**Comment by peppy** [comment]
@arflyte feedback please (leaving for the ongoing redesign is a fair call here, but make sure you consider this feedback).

**Comment by arflyte** [comment]
![3c13d06ca2](https://user-images.githubusercontent.com/3632750/36408345-329e6b3a-1648-11e8-933e-16ae740f34bd.png)

It has already been fixed in the new design, just pending for implementation.

**Comment by dorothy3242** [comment]
https://github.com/ppy/osu-web/pull/3850 looks like it was meant for this issue? though the current implementation looks strange as for the majority of usernames the set min-width is nearly 2x their length
![image](https://user-images.githubusercontent.com/33286824/47859708-42929a80-ddc5-11e8-958e-1ebf120f7978.png)
![image](https://user-images.githubusercontent.com/33286824/47859875-a026e700-ddc5-11e8-9ad1-821796f67ca9.png)

would think sinnoh's suggestion would be better for the discussion page's current design

**Comment by peppy** [comment]
It's a temporary solution until the new design.
